File: Aegis/Assets/Scripts/TriggeredAwaken.js
#pragma strict
/*
* Just drag and drop objects into the array field. Attach this to the trigger, not the object to be activated. 
*/
var hit : boolean = false;
var toEnable : GameObject[];

function OnTriggerEnter (other : Collider){
	if (other.tag == "Player"){
		hit = true;
		Debug.Log("Entered trigger "+gameObject.name);
		for (var object : GameObject in toEnable){
			object.SetActive(true);
		}
	}
}

File: Aegis/Assets/Scripts/TriggeredAwaken.cs
using UnityEngine;
using System.Collections;

public class TriggeredAwaken : MonoBehaviour {
    public GameObject[] toEnable;

    void OnTriggerEnter(Collider other)
    {
        if (other.tag == "Player")
            foreach (GameObject obj in toEnable)
            {
                if (obj != null)
                    obj.SetActive(true);
            }
    }
}
